include/picongpu/simulation/stage/CurrentInterpolationAndAdditionToEMF.hpp

```diff
@@ -78,7 +80,13 @@ namespace picongpu
                     if(kindName == "none")
                         interpolation.kind = CurrentInterpolation::Kind::None;
                     else if(kindName == "binomial")
+                    {
+                        // Current backround and binomial interpolation are incompatible #4250
+                        if(hasCurrentBackground)
```

Comment:
can be a `if constexp()` too

Reply:
Yes, and for the same reason I decided against it.